docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -18,6 +18,18 @@ Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {q
 
 The solution is located in the `getting-started-command-mode` {quickstarts-tree-url}/getting-started-command-mode[directory].
 
+== Creating the Maven project
+
+First, we need to create a new Quarkus project with the following command:
+
+:create-app-artifact-id: command-mode-quickstart
+include::{includes}/devtools/create-app.adoc[]
+
+NOTE: By default, our tooling will include the RESTEasy extension. As it is not desired for a command mode application,
+you need to add `-no-code` for the CLI or `-DnoCode` for Maven. In the same way, if you use code.quarkus.io to generate a
+project, you need to go to *MORE OPTIONS -> Starter Code* and select *No* to avoid adding the RESTEasy extension.
```

Comment:
```suggestion
NOTE: The suggested project creation command lines disable the codestarts to avoid including a REST server. Similarly, if you use code.quarkus.io to generate a
project, you need to go to *MORE OPTIONS -> Starter Code* and select *No* to avoid adding the RESTEasy Reactive extension.

The RESTEasy Reactive extension is added automatically only if you ask for codestarts and you didn't specify any extensions.
```